packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/LeftToolbar.tsx

```diff
@@ -26,43 +26,11 @@ import { Bond } from './Bond'
 import { RGroup } from './RGroup'
 import { Shape } from './Shape'
 import { Transform } from './Transform'
+import * as leftToolbarOptions from './leftToolbarOptions'
```

Comment:
Please do not use alias for star symbol: https://github.com/airbnb/javascript#modules--no-wildcard